### PR TITLE
Ignore is a list for dependabot

### PR DESCRIPTION
**Pull request by timja**
Introduced in https://github.com/jenkinsci/jenkins/pull/5120

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#ignore

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ updates:
     schedule:
       interval: "daily"
     ignore:
-      dependency-name: "org.codehaus.groovy:groovy-all"
+      - dependency-name: "org.codehaus.groovy:groovy-all"
```
Comment (MarkEWaite) [inline .github/dependabot.yml:12]: Proposing to add commons-beanutils to the list as part of this fix.  See [PR-5124](https://github.com/jenkinsci/jenkins/pull/5124) and the recommendation from @basil 
```suggestion
      - dependency-name: "org.codehaus.groovy:groovy-all"
      - dependency-name: "commons-beanutils:commons-beanutils"
```

**Review by MarkEWaite — APPROVED** [review]
Optional addition to include commons-beanutils in the exclusion list.

See https://github.com/jenkinsci/jenkins/pull/5124 and the recommendation from @basil

**Review by jglick — APPROVED** [review]
I feel like GitHub had some way of checking syntax of `dependabot.yml` automatically in PRs touching it?

**Comment by timja** [comment]
> I feel like GitHub had some way of checking syntax of `dependabot.yml` automatically in PRs touching it?

yeah normally you get a check run but I don't see it atm

**Review by basil — APPROVED** [review]

**Comment by timja** [comment]
o it's there now

File: .github/dependabot.yml [hunk 2]
```diff
@@ -9,4 +9,5 @@ updates:
...
+      - dependency-name: "commons-beanutils:commons-beanutils"
```
Comment (jglick) [inline .github/dependabot.yml:13]: BTW we should be including comments referring to written reasons for avoiding certain updates. For Groovy it is pretty well-known but for BeanUtils the issues in #4928 and #4328 were pretty technical.

Comment (timja) [reply .github/dependabot.yml:13]: just wanted to fix dependabot in this PR,

comments added in https://github.com/jenkinsci/jenkins/pull/5130 feel free to debate / suggest phrasing

**Pull request merged**